k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -465,6 +469,9 @@ public static void save(StorageEditor editor) {
         editor.putBoolean("hideUserAgent", mHideUserAgent);
         editor.putBoolean("hideTimeZone", mHideTimeZone);
 
+        editor.putString("cryptoProvider", sCryptoProvider);
```

Comment:
I would really prefer if this sort of setting referred to OpenPGP rather than 'Crypto'.